src/temp/Project/Filter.tsx

```diff
@@ -0,0 +1,107 @@
+import FilterIcon from '../icons/FilterIcon';
+import { Button } from '@mui/material';
+import style from './Filter.module.scss';
+import { useTranslations } from 'next-intl';
+import { useState } from 'react';
+
+export type Classification =
+  | 'large-scale-planting'
+  | 'agroforestry'
+  | 'natural-regeneration'
+  | 'managed-regeneration'
+  | 'urban-planting'
+  | 'other-planting';
+
+interface FilterProps {
+  filterApplied: Classification | undefined;
+  setFilterApplied: (newValue: Classification | undefined) => void;
+  availableFilters: Classification[];
+}
+
+export const FilterDropDown = ({
+  filterApplied,
+  setFilterApplied,
+  availableFilters,
+}: FilterProps) => {
+  const tAllProjects = useTranslations('AllProjects');
+
+  const handleFilterSelection = (
+    filterItem: Classification | undefined
+  ): void => {
+    setFilterApplied(filterItem);
+  };
+
+  return (
+    <>
+      {availableFilters.length > 0 ? (
+        <div className={style.projectListMainContainer}>
```

Comment:
Rename this class to `filterListContainer` or `filterOptionsContainer`, as `projectListContainer` would usually indicate a container with a list of projects.